suite-native/notifications/src/components/Notification.tsx

```diff
@@ -43,6 +43,14 @@ const notificationContainerStyle = prepareNativeStyle(utils => ({
     ...utils.boxShadows.small,
 }));
 
+const notificationContentStyle = prepareNativeStyle(_ => ({
+    flexShrink: 1,
+}));
+
+const notificationTextsStyle = prepareNativeStyle(_ => ({
+    flexShrink: 1,
+}));
```

Comment:
I prefer to divide it into two different style objects because I believe that it is more convenient for refactoring. If one of the styles is changed later (e.g. a prop is added to one of them) it might happen that you will change also a style of some else component part unintentionally. 

IMO style variable name should always reflect a specific part of the component. For example `notificationTextsStyle` sounds better to me than the general `flexShrinkBox`.

Reply:
IMO it's better not to repeat style too much. It's fine with different files but within the same file, unless it's needed during refactoring, I don't really see a reason why to duplicate code. It's the same principle as with classes and unless there is need for it (there isn't now) it's just duplication without a reason for it. 

If there is need to refactor it in the future, it's like 1 minute job max and it's really not that hard to double check where the style is used when refactoring as this is something you should do anyway.

Reply:
It would be easier to refactor callbacks without reusing them as well, why bother making them reusable then?